client/app/components/calendar/event/details/EventDetails.jsx

```diff
@@ -0,0 +1,51 @@
+import includes from 'lodash/fp/includes';
+import React from 'react';
+import EventSchema from 'schemas/EventSchema';
+
+import EventTime from './EventTime';
+import EventSummary from './EventSummary';
+import EventCreator from './EventCreator';
+import EventLocation from './EventLocation';
+import EventAttendees from './EventAttendees';
+import EventDescription from './EventDescription';
+
+import '../event.scss';
+
+const EventDetails = ({ event, timeFormat, fields }) => {
+  const time = includes('time')(fields) ? <EventTime event={event} timeFormat={timeFormat} /> : '';
+  const summary = includes('summary')(fields) ? <EventSummary event={event} /> : '';
+  const creator = includes('creator')(fields) ? <EventCreator event={event} /> : '';
+  const location = includes('location')(fields) ? <EventLocation event={event} /> : '';
+  const attendees = includes('attendees')(fields) ? <EventAttendees event={event} /> : '';
+  const description = includes('description')(fields) ? <EventDescription event={event} /> : '';
+  return (
```

Comment:
Cool! Looks much better to me 👍 Could you add one empty line between the last const declaration and return ?